runtime/zeitgeist/src/parameters.rs

```diff
@@ -80,6 +80,15 @@ parameter_types! {
     pub const TechnicalCommitteeMaxProposals: u32 = 64;
     pub const TechnicalCommitteeMotionDuration: BlockNumber = 7 * BLOCKS_PER_DAY;
 
+    // Contracts
+    pub const ContractsDeletionQueueDepth: u32 = 128;
+    pub ContractsDeletionWeightLimit: Weight = Perbill::from_percent(10) * RuntimeBlockWeights::get().max_block;
+    pub const ContractsDepositPerByte: Balance = deposit(0,1);
+    pub const ContractsDepositPerItem: Balance = deposit(1,0);
+    pub const ContractsMaxCodeLen: u32 = 123 * 1024;
```

Comment:
Not sure if this deviation from kitchensink was planned.

```suggestion
    pub const ContractsMaxCodeLen: u32 = 128 * 1024;
```

Reply:
They changed it in the lasted version, not sure why though, I just aligned the value: https://github.com/paritytech/substrate/blob/a626589bf9363b0b8659b75a1cf18a246cf1ed58/bin/node/runtime/src/lib.rs#L1242